docs/topics/object-declarations.md

```diff
@@ -230,44 +162,52 @@ data object MySingleton
 fun main() {
     val evilTwin = createInstanceViaReflection()
 
-    println(MySingleton) // MySingleton
-    println(evilTwin) // MySingleton
+    println(MySingleton) 
+    // MySingleton
+
+    println(evilTwin) 
+    // MySingleton
 
-    // Even when a library forcefully creates a second instance of MySingleton, its `equals` method returns true:
-    println(MySingleton == evilTwin) // true
+    // Even when a library forcefully creates a second instance of MySingleton, 
+    // its `.equals()` function returns true:
+    println(MySingleton == evilTwin) 
+    // true
 
-    // Do not compare data objects via ===.
-    println(MySingleton === evilTwin) // false
+    // Do not compare data objects using ===
+    println(MySingleton === evilTwin) 
+    // false
 }
 
 fun createInstanceViaReflection(): MySingleton {
     // Kotlin reflection does not permit the instantiation of data objects.
-    // This creates a new MySingleton instance "by force" (i.e. Java platform reflection)
+    // This creates a new MySingleton instance "by force" (using Java platform reflection)
     // Don't do this yourself!
     return (MySingleton.javaClass.declaredConstructors[0].apply { isAccessible = true } as Constructor<MySingleton>).newInstance()
 }
 ```
 
-The generated `hashCode()` function has behavior that is consistent with the `equals()` function, so that all runtime 
+The generated `.hashCode()` function has a behavior that is consistent with the `.equals()` function, so that all runtime
 instances of a `data object` have the same hash code.
 
 #### Differences between data objects and data classes
 
-While `data object` and `data class` declarations are often used together and have some similarities, there are some 
+While `data object` and `data class` declarations are often used together and have some similarities, there are some
 functions that are not generated for a `data object`:
 
-* No `copy()` function. Because a `data object` declaration is intended to be used as singleton objects, no `copy()` 
-  function is generated. The singleton pattern restricts the instantiation of a class to a single instance, which would 
+* No `.copy()` function. Because a `data object` declaration is intended to be used as singletons, no `.copy()`
+  function is generated. Singletons restrict the instantiation of a class to a single instance, which would
   be violated by allowing copies of the instance to be created.
-* No `componentN()` function. Unlike a `data class`, a `data object` does not have any data properties. 
-  Since attempting to destructure such an object without data properties would not make sense, no `componentN()` functions are generated.
+* No `.componentN()` function. Unlike a `data class`, a `data object` does not have any data properties.
+  Since attempting to destructure such an object without data properties would not make sense, no `.componentN()` functions are generated.
```

Comment:
```suggestion
* No `componentN()` function. Unlike a `data class`, a `data object` does not have any data properties.
  Since attempting to destructure such an object without data properties would not make sense, no `componentN()` functions are generated.
```